introduction.md

```diff
@@ -0,0 +1 @@
+Hi, my name is Ning Wang. I'm come from Taiwan, and I think coding is a little bit too easy for me.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.